src/NServiceBus.Core/Pipeline/Incoming/TransportReceiveContext.cs

```diff
@@ -31,9 +29,9 @@ public TransportReceiveContext(IncomingMessage receivedMessage, TransportTransac
         /// </summary>
         public void AbortReceiveOperation()
         {
-            cancellationTokenSource.Cancel();
+            ReceiveOperationWasAborted = true;
         }
 
-        CancellationTokenSource cancellationTokenSource;
+        internal bool ReceiveOperationWasAborted { get; private set; }
```

Comment:
I think this should be `public`. The type is `internal` and its other members are `public`.

Reply:
This was intentional to hide it from users, do U see a use case where user code should see this?

Reply:
If the type is internal it is already hidden no?

Reply:
You are right, will make it public!